packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -227,16 +333,51 @@ class _BottomNavigationTile extends StatelessWidget {
     // produce smooth animation. We do this by multiplying the flex value
     // (which is an integer) by a large number.
     int size;
-    Widget label;
+
+    /// Defines the padding for the animating icons + labels.
```

Comment:
Would these comments be useful on the overall class, here they won't be seen in dartdocs?

Reply:
To be honest I'm not sure they're useful as overall dartDocs. The developers using `BottomNavigationBar`s typically don't care _how_ the padding is calculated, just how to use it. It is meant more for people working on this class in the future to know how the padding is calculated.

Reply:
Understood, in that case I would use `//` comments

Reply:
Fixed.